src/index.html

```diff
@@ -9,10 +9,39 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="../src/images/imac.jpeg"
```

Comment:
Please verify that the path to the image (`../src/images/imac.jpeg`) is correct and that the image exists in the specified directory. This is crucial for the image to be displayed properly on the webpage.